WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupUtils.kt

```diff
@@ -0,0 +1,43 @@
+package org.wordpress.android.ui.accounts.signup
+
+import java.util.Locale
+import javax.inject.Inject
+
+class SignupUtils
+@Inject constructor() {
+    /**
+     * Create a display name from the email address by taking everything before the "@" symbol,
+     * removing all non-letters and non-periods, replacing periods with spaces, and capitalizing
+     * the first letter of each word.
+     *
+     * @return [String] to be the display name
+     */
+    fun createDisplayNameFromEmail(emailAddress: String): String? {
+        val username = emailAddress.split("@").firstOrNull() ?: return null
+        val usernameArray = username.replace("[^A-Za-z/.]".toRegex(), "").split(".")
+        val builder = StringBuilder()
+        for (item in usernameArray) {
+            if (item.isNotEmpty()) {
+                builder.append(item.substring(0, 1).toUpperCase(Locale.ROOT))
+                if (item.length > 1) {
+                    builder.append(item.substring(1))
+                }
+                builder.append(" ")
+            }
+        }
+        val displayName = builder.toString().trim { it <= ' ' }
```

Comment:
Unless I'm missing something, I think `trim` can be simplified here:

```suggestion
        val displayName = builder.toString().trim()
```

Reply:
I think you're right. This is a result of the automatic conversion to kotlin